src/current/_includes/releases/v23.2/v23.2.4.md

```diff
@@ -0,0 +1,56 @@
+## v23.2.4
+
+Release Date: April 9, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-4-sql-language-changes">SQL language changes</h3>
+
+- Mutation statements such as [`UPDATE`](../v23.2/update.html) and [`DELETE`](../v23.2/delete.html) as well as locking statements such as [`SELECT FOR UPDATE`](../v23.2/select-for-update.html) are not allowed in [read-only transactions](../v23.2/set-transaction.html#parameters) or [`AS OF SYSTEM TIME` transactions](../v23.2/set-transaction.html#parameters). Previously, a bug existed where mutation statements and locking statements in implicit single-statement transactions using AS OF SYSTEM TIME were allowed. [#120158][#120158]
+- The new cluster setting [`sql.stats.virtual_computed_columns.enabled`](../v23.2/cluster-settings.html#setting-sql-stats-virtual-computed-columns-enabled), when enabled, allows the collection of [table statistics](../v23.2/show-statistics.html) on [virtual computed columns](../v23.2/computed-columns.html). [#120933][#120933]
+- The new [session variable](../v23.2/session-variables.html) `optimizer_use_virtual_computed_column_stats`, when enabled, configures the [cost-based optimizer](../v23.2/cost-based-optimizer.html) to use [table statistics](../v23.2/show-statistics.html) on [virtual computed columns](../v23.2/computed-columns.html). [#121329]
+
+<h3 id="v23-2-4-db-console-changes">DB Console changes</h3>
+
+- Fixed an issue where clusters with multiple [stores](../v23.2/cockroach-start.html#store) per node could list inaccurate region and node information on the [**Databases** page](../v23.2/ui-databases-page.html#databases). [#120212][#120212]
+- Users will no longer see [views](../v23.2/views.html) displayed on the [**Databases** page](../v23.2/ui-databases-page.html#databases). Previously views would be listed with no information, only displaying errors. [#120214][#120214]
+
+<h3 id="v23-2-4-bug-fixes">Bug fixes</h3>
+
+- Previously, on long-running [sessions](../v23.2/show-sessions.html) that issue many (hundreds of thousands or more) [transactions](../v23.2/transactions.html), CockroachDB's internal memory accounting system, the limit for which is configured via the [`--max-sql-memory` flag](../v23.2/cockroach-start.html#general)) could leak. This bug, in turn, could result in the error message `"root: memory budget exceeded"` for other queries. The bug was present in v23.1.17 and is now fixed. [#121875][#121875]
```

Comment:
The bug is also present on 23.2.3 version (i.e. only specifically 23.1.17 and 23.2.3). (I did mention this in the original release note to https://github.com/cockroachdb/cockroach/pull/121875, but I think the mention of 23.2.3 version was consciously omitted from 23.1.18 release notes.)